tools/riemann-elasticsearch/bin/riemann-elasticsearch

```diff
@@ -49,6 +50,14 @@ class Riemann::Tools::Elasticsearch
     "http://#{options[:es_host]}:#{options[:es_port]}#{path_prefix.length>0?'/':''}#{path_prefix}/_stats/index,store"
   end
 
+  def search_url
+    es_search_index = options[:es_search_index]
+    path_prefix = options[:path_prefix]
+    path_prefix[0] = '' if path_prefix[0]=='/'
+    path_prefix[path_prefix.length-1] = '' if path_prefix[path_prefix.length-1]=='/'
```

Comment:
This is essentially duplicate code - perhaps pull out into a method and call it in all the `*_url` methods?